releases/3.3.0/artifacts.json

```diff
@@ -0,0 +1,242 @@
+{
+  "3.3.0": {
+    "holoscan": {
+      "debian-version": "3.3.0-1",
+      "wheel-version": "3.3.0",
+      "base-images": {
+        "dgpu": "nvcr.io/nvidia/cuda:12.8.1-runtime-ubuntu24.04",
+        "igpu": "nvcr.io/nvidia/tensorrt:25.03-py3-igpu"
+      },
+      "build-images": {
+        "igpu": {
+          "jetson-agx-orin-devkit": "nvcr.io/nvstaging/holoscan/holoscan:v3.3.0.0-igpu",
+          "igx-orin-devkit": "nvcr.io/nvstaging/holoscan/holoscan:v3.3.0.0-igpu",
+          "sbsa": "nvcr.io/nvstaging/holoscan/holoscan:v3.3.0.0-igpu"
+        },
+        "dgpu": {
+          "x64-workstation": "nvcr.io/nvstaging/holoscan/holoscan:v3.3.0.0-dgpu",
+          "igx-orin-devkit": "nvcr.io/nvstaging/holoscan/holoscan:v3.3.0.0-dgpu",
+          "sbsa": "nvcr.io/nvstaging/holoscan/holoscan:v3.3.0.0-dgpu",
+          "clara-agx-devkit": "nvcr.io/nvstaging/holoscan/holoscan:v3.3.0.0-dgpu"
+        },
+        "cpu": {
+          "x64-workstation": "nvcr.io/nvstaging/holoscan/holoscan:v3.3.0.0-dgpu"
+        }
+      }
+    },
+    "health-probes": {
```

Comment:
Would the health probes ever be specific to a given hsdk version?

Reply:
It is, every version has its own set of links.